source/_reference-g-s.html.md.erb

```diff
@@ -1274,6 +1274,1116 @@ Imports CSV or XLSX data into a new sheet in the specified workspace.
 
 <aside class="notice">Note the following:<ul><li>Both <b>sheetName</b> and the file name must use ASCII characters.</li><li>The source data must be basic text. To include rich formula data, import and create a sheet first, and then use <a href="#update-rows">Update Rows</a>. To work with images, see <a href="#cell-images">Cell Images</a>.</li><li>XLS is not supported. You must use XLSX.</li><li>Hierarchical relationships between rows in an external file won't import.</li></ul></aside>
 
+# Proofs
+
+A proof is a container that holds at least one attachment or comment. Limited to one proof and its versions per row. A sheet can have multiple proofs. 
+
+## <%= OBJECTS %>
+
+### Proof Object
+
+> Example: Proof object
+
+```json
+{
+	"id": 0,
+	"originalId": 0,
+	"name": "string",
+	"proofType": "DOCUMENT",
+	"proofRequestUrl": "string",
+	"version": 0,
+	"lastUpdatedAt": "2019-08-24T14:15:22Z",
+	"lastUpdatedBy": {
+		"email": "jane.doe@smartsheet.com",
+		"name": "Jane Doe"
+	},
+	"isCompleted": true,
+	"attachments": [{}],
+	"discussions": [{}]
+}
+```
+
+     |     |     |
+-----|-----|-----|
+**id** | number | Proof Id of the latest proof version. |
+**originalId** | number | Proof Id of the original proof version. |
+**attachments** | [Attachment](#attachment-object)[] | Array of Attachment objects. Only returned if the include query string parameter contains attachments. |
+**discussions** | [Discussion](#discussion-object)[] | Array of Discussion objects. Only returned if the include query string parameter contains discussions. |
+**isCompleted** | Boolean | Indicates whether the proof is completed. |
+**lastUpdatedAt** | timestamp | A timestamp when the proof was last updated. |
+**lastUpdatedBy** | [User](#user-object) | User object containing name and email of the user who last updated the proof. |
```

Comment:
I assumed that was just to fix the OpenAPI model rather than something to surface to customers. Though We could change to MiniUser and expose that to the public. I could go either way on this one.